lib/OpenQA/Git.pm

```diff
@@ -81,6 +81,20 @@ sub set_to_latest_master ($self, $args = undef) {
         return $self->_format_git_error($res, 'Unable to reset repository to origin/master') unless $res->{status};
     }
 
+    if ((!$self->config->{update_remote}) && (!$self->config->{update_branch})) {
+        eval {
```

Comment:
We probably want to use `Feature::Compat::Try` in new code.

Reply:
The file `Git.pm` already using it https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/Git.pm#L12